speech/snippets/speech_model_adaptation_beta.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2024 Google LLC
```

Comment:
Please roll back this! Copyright is fixed, on the date the file is created.

Reply:
Lets rollback the changes for this file! Existing scripts looks good to me at https://cloud.google.com/speech-to-text/docs/adaptation